README.md

```diff
@@ -0,0 +1,220 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [awesomebooks] <a name="about-project"></a>
+
+>
+
+**[awesomebooks]** This is awesome books website, where you register a book with author's name. It also allows you to perform CRUD operations on it.
+
+## 🛠 Built With <a name="built-with">HTML, CSS and javascript</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech stack
+> I have used vanilla js and luxon.js to build this project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://css.org/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Interactivity</summary>
+  <ul>
+    <li><a href="https://js.com/">vanilla js</a></li>
+  </ul>
+</details>
+
+<details>
```

Comment:
- [ ] Please consider removing this `<details>` tag to make your content visible, at the moment, it is hidden on a "details" dropdown list. 🤓 🛠️   


https://user-images.githubusercontent.com/77069011/206855762-cc8c177d-2cb3-4e47-acbe-40d650975992.mp4